# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../uclibc/pkg_patch/php-uts_domainname.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Taken from https://dev.openwrt.org/browser/packages/lang/php5/patches/02-uts_domainname.patch?rev=6476

diff -urN php-5.0.5/ext/posix/posix.c php-5.0.5.new/ext/posix/posix.c
--- php-5.0.5/ext/posix/posix.c	2005-06-07 00:06:00.000000000 +0200
+++ php-5.0.5.new/ext/posix/posix.c	2007-03-03 12:14:41.000000000 +0100
@@ -438,7 +438,7 @@
 	add_assoc_string(return_value, "version",  u.version,  1);
 	add_assoc_string(return_value, "machine",  u.machine,  1);
 #ifdef _GNU_SOURCE
-	add_assoc_string(return_value, "domainname", u.domainname, 1);
+	add_assoc_string(return_value, "domainname", u.__domainname, 1);
 #endif
 }
 /* }}} */
